README.md

```diff
@@ -0,0 +1,80 @@
+# OpenTelemetry Instrumentation: Spring and SpringBoot
+
```

Comment:
Add a doc comment here here that states this is an in-progress draft to be contributed in multiple PRs.
Also add a clear statement saying this document is in progress.